src/main/java/Duke.java

```diff
@@ -1,10 +1,237 @@
+        // Prints enclosed text in italics
+        String byline = "\033[3m----------------------------------Just a rather very intelligent system---------\033[0m\n";
+        System.out.println(byline + "Good Evening Sir! I'm J.A.R.V.I.S");
+        System.out.println("How may I be of assistance to you today? " + "\uD83D\uDE01");
+        System.out.println(LINE + System.lineSeparator());
+    }
+
+    public static void printBye() {
+        System.out.println(LINE + System.lineSeparator() + "Affirmative sir, I'll shut down all operations"
+                + System.lineSeparator() + LINE);
+    }
+
+    public static void printList() {
+        System.out.println(LINE + System.lineSeparator() + "Here are the current tasks in your list:");
+        for (Task task : tasks) {
+            System.out.println(tasks.indexOf(task) + 1 + "." + task.printTask());
+        }
+        System.out.println(LINE);
+    }
+
+    public static void printListSummary(Task task) {
+        System.out.println(LINE + System.lineSeparator() + "Will do sir, I've added: "
+                    + System.lineSeparator() + "  " + task.printTask());
+        if (tasks.size() == 1) {
+            System.out.printf("Now you have %d task in your list.\n", tasks.size());
+        } else {
+            System.out.printf("Now you have %d tasks in your list.\n", tasks.size());
+        }
+        System.out.println(LINE);
+    }
+
+    public static void markAsDone(String command) {
+        // When user enters string "done 2", string is split to extract the index 2 only
+        int taskDoneIndex = Integer.parseInt(String.valueOf(command.substring(DONE_SIZE))) - 1;
+                //Integer.parseInt(command.split(" ")[1]) - 1;
+        // Checks if given index holds a task and throws error message if no such task exists
+        if (taskDoneIndex >= tasks.size() || taskDoneIndex < 0) {
+            System.out.println(LINE + System.lineSeparator() +
+                    "Apologies sir but, it seems that task hasn't been created yet " +
+                    "\u2639" + System.lineSeparator() + LINE);
+        } else {
+            // Selects task to be modified with command "done"
+            Task taskChosen = tasks.get(taskDoneIndex);
+            // Checks if task has already been marked as done
+            if (taskChosen.isDone()) {
+                System.out.println(LINE + System.lineSeparator() + "Sir, I believe this task has already been completed "
+                    + "\uD83E\uDD14");
+            } else {
+                taskChosen.changeStatusDone(true);
+                System.out.println(LINE + System.lineSeparator() + "As you wish sir, this task will be marked as done! "
+                    + "\uD83D\uDE01");
+            }
+            // Otherwise, marks task as done with X. E.g. 1.[ ][X] read book if user inputs "done 1"
+            System.out.println("    " + taskChosen.printTask() + System.lineSeparator() + LINE);
+        }
+    }
```

Comment:
Note if some of the comments are actually necessary, if there are not do remove them. Applies to other parts of your code as well